provisioner/salt-masterless/provisioner.go

```diff
@@ -398,6 +398,32 @@ func (p *Provisioner) Provision(ctx context.Context, ui packer.Ui, comm packer.C
 		}
 	}
 
+	if p.config.GuestOSType == provisioner.WindowsOSType {
+		{
+			ui.Message("Running salt-call --local winrepo.update_git_repos")
```

Comment:
So I just hooked this up on a simple windows instance to try and do a very basic salt test; got this error: 

```==> amazon-ebs: Error running 'winrepo.update_git_repos': Git for Windows is not installed, or not configured to be accessible from the Command Prompt```

I don't think we can assume that all users of Salt have git installed, can we?

Reply:
Sorry, I meant to add a comment regarding the Git dependency.  I had considered downloading and installing Git as part of the provisioner initialization process but wasn't sure if it was preferred that a dependency like this be documented as a prerequisite or incorporated into the provisioner.  In my view the later is a better approach as it makes the provisioner self-contained and easier to use.  The downside is the additional maintenance of keeping the Git client up to date.  I went ahead and made this change and updated the pull request.  Let me know what you think.